cmd/proxy.go

```diff
@@ -58,13 +57,13 @@ func NewProxyCmd() *cobra.Command {
 				return fmt.Errorf("building kangal clientset: %w", err)
 			}
 
-			kubeClientSet, err := kubernetes.NewForConfig(k8sConfig)
+			kubeClientSet, err := kubeClient.NewForConfig(k8sConfig)
 			if err != nil {
 				return fmt.Errorf("building kubernetes clientset: %w", err)
 			}
 
 			loadTestClient := kangalClientSet.LoadTests()
-			kubeClient := kube.NewClient(loadTestClient, kubeClientSet, logger)
+			kubeClient := kubernetes.NewClient(loadTestClient, kubeClientSet, logger)
```

Comment:
Maybe it's better to rename the alias `kubeClient` to `kubernetesClient`? This way you don't have a variable and import named `kubeClient`

Reply:
Done via 97b1771